GitUI/UserControls/RevisionGrid/RevisionDataGridView.cs

```diff
@@ -476,10 +477,6 @@ public void LoadingCompleted()
                 })
                 .FileAndForget();
             }
-            else
-            {
-                MarkAsDataLoadingComplete();
-            }
```

Comment:
Could you please explain why this is being removed?

Reply:
It is moved to line 430

Reply:
Thank you, somehow I missed that.